internal/service/controller_service.go

```diff
@@ -105,6 +105,8 @@ func (s *ControllerService) Register(ctx context.Context, req *pb.RegisterReques
 		return nil, err
 	}
 
+	patch := client.StrategicMergeFrom(exporter.DeepCopy())
```

Comment:
```suggestion
	original := client.StrategicMergeFrom(exporter.DeepCopy())
```

Suggestion,  this way it makes more sense if you are not familiar with how "Patch(ctx, new, original)" works ?

Reply:
at least I was confused at first read :)